lib/hexpm_web/views/icons.ex

```diff
@@ -1,135 +1,43 @@
 defmodule HexpmWeb.ViewIcons do
   use Phoenix.HTML
-  import SweetXml
 
   @icons_dir Path.join(__DIR__, "../../../assets/vendor/icons")
-  @octicons_svg Path.join(@icons_dir, "octicons.svg")
-  @glyphicons_svg Path.join(@icons_dir, "glyphicons-halflings-regular.svg")
-  @glyphicons_less Path.join(@icons_dir, "glyphicons.less")
+  @heroicons_svg Path.join(@icons_dir, "heroicons.svg")
```

Comment:
Let's add a comment with a link to https://github.com/supersimple/HeroiconsToLib so we know how to regenerate this file.